src/services/UserService.ts

```diff
@@ -102,7 +100,7 @@ export class UserService {
 
 		const user = await getRepository(User).findOne({ email });
 		if (!user) {
-			throw new APIError(404, AuthenticateError.AccountNotFound);
+			throw new APIError(400, AuthenticateError.AccountNotFound);
```

Comment:
Don't have to change this here, but it may be a good idea to create a enum for these status codes. It might help the code a bit more readable for someone that isn't familiar with what the HTTP response codes mean 🙂

Reply:
I think this would make a good first issue for someone to have a go at doing!